src/makeCalculator.js

```diff
@@ -37,7 +37,43 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    reset() {
+      this.result = 0;
+
+      return this;
+    },
+
+    add(operand1, operand2) {
+      return operand1 + operand2;
+    },
+
+    subtract(operand1, operand2) {
+      return operand1 - operand2;
+    },
+
+    divide(operand1, operand2) {
+      if (operand1 === 0) {
+        return;
+      }
```

Comment:
what if your operand2 will be 0?
we can divide 0 by something, but we can't divide something by 0